experiments/explore-research-outputs/notebooks/utils.py

```
@@ -0,0 +1,129 @@
+def fetch_api_swagger_json(url, save_to_filepath=None) -> object:
+    """
+    Attempts to fetch json from the provided endpoint.
+
+    Arguments:
+    url: url to the remote swagger.json file
+    save_to_filepath (optional): Relative path (where to save json file)
+
+    Returns: Response object (if successful)
+    """
+
+    response = requests.get(url)
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)